src/Ledger/Fees.lagda

```diff
@@ -7,62 +7,74 @@ open import Ledger.Transaction
 
 module Ledger.Fees
   (txs : _) (open TransactionStructure txs)
-  -- (abs : AbstractFunctions txs) (open AbstractFunctions abs)
   where
 
 open import Data.Rational using (0ℚ; ℚ; mkℚ+; _*_; floor)
-open import Data.Nat.Coprimality using (1-coprimeTo)
+open import Data.Rational.Literals using (number)
 open import Data.Nat.Induction using (<′-wellFounded)
-open import Data.Nat.Properties using (<⇒<′; ≰⇒>)
+open import Data.Nat.Properties using (≤′-trans; <⇒<′; ≰⇒>)
 open import Data.Integer using (∣_∣)
-open import Data.Product using (swap)
 open import Induction.WellFounded using (Acc; acc)
+open import Agda.Builtin.FromNat using (Number)
+
+open Number number renaming (fromNat to fromℕ)
 \end{code}
 
 \section{Fee Calculation}
 \begin{figure*}[h]
 \begin{AgdaMultiCode}
-\begin{code}[hide]
-_↑ℚ : ℕ → ℚ
-n ↑ℚ = mkℚ+ n 1 (1-coprimeTo n ∘ swap)
-\end{code}
 \begin{code}
 scriptsCost : (pp : PParams) → ℕ → Coin
```

Comment:
Why is the name in plural? If I understand correctly this is used to calculate the cost of one script

Reply:
No, the argument it gets is the sum of the sizes of reference scripts, so this name is correct. Is there some inconsistency elsewhere?

Reply:
Somewhere in the explanation it was suggested:
"... which returns an integer that is the total size of the reference script ..."

Reply:
That may be my fault.  I suppose it should say something like, "the sum total of the sizes of the reference scripts."  @WhatisRT could you confirm?

Reply:
Yeah, the explanation should be adjusted then